src/components/Browse/BrowseResults.js

```diff
@@ -226,7 +226,7 @@ export class BrowseResults extends React.Component<Props, S> {
                         id="refType1"
                         style={{ fontWeight: 'bold', margin: 0, padding: 0 }}
                       >
-                        {'See: '}
+                        See:
```

Comment:
This should be added to the translation file `en.json`.

Reply:
I have included the texts in en.json